.changeset/khaki-numbers-nail.md

```diff
@@ -0,0 +1,7 @@
+---
+'@firebase/auth': minor
```

Comment:
Why a minor for auth? The API doesn't change. I don't think we bump automatically based on dependencies' bump level. I'm not totally sure if changesets does it but I think it also doesn't.

Reply:
Ok, switched Auth to a patch bump instead of a minor bump.